shotover-proxy/src/frame/cassandra.rs

```diff
@@ -471,38 +503,497 @@ impl CassandraOperation {
 }
 
 #[derive(PartialEq, Debug, Clone)]
-pub enum CQL {
-    // Box is used because Statement is very large
-    Parsed(Box<Statement>),
-    FailedToParse(String),
+pub struct CQLStatement {
+    pub statement: CassandraStatement,
+    pub has_error: bool,
```

Comment:
same as other has_error field.

Reply:
With has_error removed can we just replace this struct with CassandraStatement entirely